kubernetes-examples/envelope-encryption/README.md

```diff
@@ -0,0 +1,83 @@
+# Envelope Encryption Example on Kubernetes.
+
+This example will demonstrate the `EnvelopeEncryption` filter which provides an Encryption-at-Rest solution for the
+Apache Kafka(tm) which is transparent to both clients and brokers.
+
+The `EnvelopeEncryption` filter works by intercepting all produce requests from applications and applies encryption to
+the kafka records as they pass through Kroxylicious. On the consume path, the reverse happens - the filter intercepts the fetch responses and decrypts the records before they are sent to the application.
+
+
+## What will the example demonstrate?
+
+It will:
+
+* deploy an Apache Kafka Cluster to Kubernetes using Strimzi.
+* deploy an instance of Kroxylicious to the same Kubernetes cluster  to proxy the Kafka Cluster with configuration for
+  `EnvelopeEncryption`.
+* deploy an instance of HashiCorp Vault to store the keys used to encrypt the kafka records.
+* use an off-cluster kafka producer and kafka consumer send and receive messages, and demonstrate
+  that the records are indeed encrypted on the server.
+
+Running the example will take about 10 minutes.
+
+## Prerequisites
+
+See [prerequistes](../README.md#prerequisites-to-run-the-kubernetes-examples).
+
+## Running the example
+
+1. Clone Kroxylicious Repository
+    ```shell { adjunct="# We're going to demonstrate the Envelope Encryption feature of Kroxylicious.Let's start by cloning the repo." }
```

Comment:
nit!!
```suggestion
    ```shell { adjunct="# We're going to demonstrate the Envelope Encryption feature of Kroxylicious. Let's start by cloning the repo." }
```